src/reference/interactive/index.pug

```diff
@@ -159,16 +166,16 @@ block reference
     p Interested in making an Interactive Project? Let's get started!
 
     h3#prerequisites Prerequisites
-    p Before getting started, you should have the following things:
+    p Before you begin, we recommend that you have:
 
     ul
-        li A Beam user account.
-        li Some knowledge of programming, unless you're using an existing Game Client or Interactive Application.
-        li An awesome idea.
+        li A Beam user account
+        li Some knowledge of programming, unless you're using an existing Game Client or Interactive Application
+        li An awesome idea
 
-    h3#choosing-an-sdk-environment Choosing an SDK / Environment
+    h3#choosing-an-sdk-environment Choose a SDK
 
-    p If you're making a Game Client from scratch, you'll need to pick an Interactive SDK.
+    p If you're making a Game Client from scratch, you'll need to pick a SDK.
```

Comment:
We want to leave interactive here, Beam has other SDKs for non interactive features